### PR TITLE
(MODULES-4098) Sync more files

### PR DESCRIPTION
**Pull request by hunner**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
 sudo: false
 language: ruby
-rvm:
-  - 2.2
+rvm: 2.3.1
 cache: bundler
-# Run msync but in noop (Test Command).  Also, do not update the sqlserver module (regex escaped for bash and then escaped for yaml)
-script: "bundle exec msync update --noop --git-base=https://github.com/ -f \\^\\(\\(\\?\\!-sqlserver\\).\\)\\*\\$"
+# Run msync but in noop (Test Command).  Also, do not update the sqlserver/vsphere modules (regex escaped for yaml)
+script: 'bundle exec msync update --noop --git-base=https://github.com/ -x "\(sqlserver\|vsphere\)"'
```
Comment (ferventcoder) [inline .travis.yml]: 👍

File: moduleroot/CONTRIBUTING.md [hunk 2]
```diff
@@ -178,8 +178,8 @@ installed on your system.
 You can run them by issuing the following command
 
 ```shell
-% rake spec_clean
-% rspec spec/acceptance
+% bundle exec rake spec_clean
+% bundle exec rspec spec/acceptance
```
Comment (ferventcoder) [inline moduleroot/CONTRIBUTING.md]: 👍

File: moduleroot/spec/acceptance/nodesets/docker/debian-8.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  debian-8-x64:
+    platform: debian-8-amd64
+    hypervisor: docker
+    image: debian:8
+    docker_preserve_image: true
+    docker_cmd: '["/sbin/init"]'
+    docker_image_commands:
+      - 'apt-get update && apt-get install -y net-tools wget locales strace lsof && echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && locale-gen'
+CONFIG:
+  trace_limit: 200
```
Comment (ferventcoder) [inline moduleroot/spec/acceptance/nodesets/docker/debian-8.yml]: These are all typically ignored on the Windows side, correct?

**Comment by ferventcoder** [comment]
LGTM

File: moduleroot/spec/acceptance/nodesets/docker/debian-8.yml (hunk 3 above)
Comment (DavidS) [inline moduleroot/spec/acceptance/nodesets/docker/debian-8.yml]: Yeah, those docker nodesets are used on travis. At one point we might want to re-use them to prove that facts and plugins do not kill agents on unsupported platforms (or when dependencies are missing).

**Pull request merged**